### ISSUE TITLE
Have you tested SVIT in some traditional tasks, e.g., VQAv2?

### ISSUE THREAD

**Issue by HenryHZY** [BAAI-DCAI/Visual-Instruction-Tuning#11]
Hi @BoyaWu10, thanks for your great project.
I find it difficult to get more useful information about SVIT from just a few images from the paper.
Have you tested SVIT in some traditional tasks, e.g., VQAv2 or GQA? I think these experiments shouldn't take too much time. Or if it's possible, you could release the checkpoint and I can conduct some experiments.
Thank you very much.

**Comment by BAAI-DCAI** [comment]
Hi, we will update the paper with more experiments and release the checkpoint in next week.

**Comment by XenonLamb** [comment]
I'm trying to train LLaVA1.5 on this finetuning set, but found the training loss notably higher than the llava 665k dataset. Can you help release the finetuning pipeline and checkpoints? Thank you!

**Comment by BoyaWu10** [comment]
Hi @HenryHZY, sorry for the late reply. Thanks for your interest in this work. We've updated our [paper](https://arxiv.org/abs/2307.04087) with more experimental results, including traditional task like VQAv2 and GQA. The checkpoints have been released ([full](https://huggingface.co/Isaachhe/svit-v1.5-13b-full/tree/main), [lora](https://huggingface.co/Isaachhe/svit-v1.5-13b-lora/tree/main)).

**Comment by BoyaWu10** [comment]
> I'm trying to train LLaVA1.5 on this finetuning set, but found the training loss notably higher than the llava 665k dataset. Can you help release the finetuning pipeline and checkpoints? Thank you!

Hi @XenonLamb, the reason that the loss is higher is probably due to the fact that responses in SVIT are much longer than traditional datasets, such as VQAv2 and GQA, which only contain few words in the responses.

For LLaVA-v1.5 training, we have uploaded SVIT in LLaVA-v1.5's format [here](https://huggingface.co/datasets/BAAI/SVIT/tree/main/format/llava-v1.5). More details could be found in the updated [paper](https://arxiv.org/abs/2307.04087). The checkpoints trained on LLaVA-v1.5's architecture have been released ([full](https://huggingface.co/Isaachhe/svit-v1.5-13b-full/tree/main), [lora](https://huggingface.co/Isaachhe/svit-v1.5-13b-lora/tree/main)).

**Comment by HenryHZY** [comment]
Thanks for your reply!
